src/taskgraph/actions/util.py

```diff
@@ -32,8 +32,9 @@ def get_parameters(decision_task_id):
     return get_artifact(decision_task_id, "public/parameters.yml")
 
 
-def fetch_graph_and_labels(parameters, graph_config):
-    decision_task_id = find_decision_task(parameters, graph_config)
+def fetch_graph_and_labels(parameters, graph_config, decision_task_id=None):
+    if decision_task_id is None:
+        decision_task_id = find_decision_task(parameters, graph_config)
```

Comment:
One bit I'm not sure about is whether the `task_group_id` passed to the action is ever `None`, and thus whether we still need this `find_decision_task` fallback?

Reply:
Mmm, good point, I think it's probably always populated